lib/Controller/SubsonicController.php

```diff
@@ -1067,17 +1071,64 @@ protected function deleteBookmark(string $id) : array {
 	/**
 	 * @SubsonicAPI
 	 */
-	protected function getPlayQueue() : array {
-		// TODO: not supported yet
-		return ['playQueue' => []];
+	protected function getPlayQueue() : Response {
+		/** @var array|false $playQueue */
+		$playQueue = json_decode($this->configManager->getUserValue($this->user(), $this->appName, 'play_queue', 'false'), true);
+
+		if (!$playQueue) {
+			return $this->subsonicResponse([]);
```

Comment:
I'm returning an empty response rather than an empty playqueue when we haven't saved one. Seems better than generating metadata for a canned response... but I couldn't find anything in the API docs regarding how to handle this situation, so maybe it's noncompliant